app/src/main/java/com/example/parkingsystemjava/mvp/view/ReservationView.java

```diff
@@ -1,25 +1,66 @@
 package com.example.parkingsystemjava.mvp.view;
 
+import android.content.Context;
+import android.widget.Toast;
 import androidx.fragment.app.FragmentManager;
+import com.example.parkingsystemjava.R;
 import com.example.parkingsystemjava.activity.ReservationActivity;
 import com.example.parkingsystemjava.databinding.ActivityReservationBinding;
 import com.example.parkingsystemjava.fragment.DateTimeReservationDialogFragment;
 import com.example.parkingsystemjava.mvp.contract.ReservationActivityContract;
 import com.example.parkingsystemjava.mvp.view.base.ActivityView;
 import com.example.parkingsystemjava.utils.ConstantUtils;
+import com.example.parkingsystemjava.utils.DateUtils;
+import java.util.Calendar;
 
 public class ReservationView extends ActivityView implements ReservationActivityContract.ViewContract {
-    private ActivityReservationBinding binding;
+    private final ActivityReservationBinding binding;
 
     public ReservationView(ActivityReservationBinding binding, ReservationActivity reservationActivity) {
         super(reservationActivity);
         this.binding = binding;
     }
 
     @Override
-    public void showDateTimeInfo() {
+    public String getParkingLots() {
+        return binding.editTextParkingPlaceActivityReservation.getText().toString();
+    }
+
+    @Override
+    public String getPass() {
+        return binding.editTextActivityReservationCodePlace.getText().toString();
+    }
+
+    @Override
+    public void showDateTimeInfo(boolean isStartDate) {
         FragmentManager fragmentManager = getFragmentManager();
-        DateTimeReservationDialogFragment datePickerDialogFragment = new DateTimeReservationDialogFragment();
-        datePickerDialogFragment.show(fragmentManager, ConstantUtils.DIALOG_FRAGMENT_RESERVATION_TAG);
+        DateTimeReservationDialogFragment datePickerDialogFragment = DateTimeReservationDialogFragment.newInstance(isStartDate);
+        if (fragmentManager != null) {
+            datePickerDialogFragment.show(fragmentManager, ConstantUtils.DIALOG_FRAGMENT_RESERVATION_TAG);
+        }
+    }
+
+    @Override
+    public void showMessageOfSavedStatement(Calendar startDate, Calendar endDate, String parkingLots, String codePlace) {
+        Context context = getContext();
+        if (context != null) {
+            Toast.makeText(context, context.getString(R.string.toast_saved_reservation_activity, DateUtils.formatDateToString(startDate), DateUtils.formatDateToString(endDate), parkingLots, codePlace), Toast.LENGTH_LONG).show();
+        }
```

Comment:
pls identify all the toast and put into a generic method

Reply:
Done, I deleted the other toast, I'm going to put into a generic method for the PR with validations when I need to show the errors messages